### PR TITLE
Dat 4009

### PR DESCRIPTION
**Pull request by RafLeszczynski**
DO NOT merge without APP part!!! https://github.com/Wikia/app/pull/10070

## Links

* https://wikia-inc.atlassian.net/browse/DAT-4009

## Description

fix issue with go to source page redirects

## Reviewers

@SebastianMarzjan @rogatty

### CODE REVIEW

File: front/main/app/models/infobox-builder.js [hunk 1]
```diff
@@ -322,7 +322,7 @@ const InfoboxBuilderModel = Ember.Object.extend({
 						method: 'POST',
 						success: (data) => {
 							if (data && data.success) {
```
Comment (SebastianMarzjan) [inline front/main/app/models/infobox-builder.js]: "This shouldn't happen" (TM)
But what if urls are not in data?

Comment (dianafa) [inline front/main/app/models/infobox-builder.js]: nothing happens:

`			return model.saveStateToTemplate().then((urls = {}) => {
				if (shouldRedirectToPage) {
					this.get('target').send('redirectToPage', urls.templatePageUrl);
				}
			});`

that means we get redirected to the previous page:

`redirectToPage(url) {
			const action = url ?
				'redirectToPage' :
				'redirectToPreviousPage';`

File: front/main/app/routes/infobox-builder.js [hunk 2]
```diff
@@ -282,5 +279,35 @@ export default Ember.Route.extend(ConfirmationMixin, {
...
+				url: M.buildUrl({
+					path: '/wikia.php'
+				}),
+				data: {
+					controller: 'PortableInfoboxBuilderController',
+					method: 'getRedirectUrls',
+					title
+				},
+				dataType: 'json',
+				method: 'POST',
```
Comment (SebastianMarzjan) [inline front/main/app/routes/infobox-builder.js]: POSTing to a get.* method?

File: front/main/app/models/infobox-builder.js (hunk 1 above)
Comment (SebastianMarzjan) [inline front/main/app/models/infobox-builder.js]: Cool, didn't notice the default value. Thanks!

**Comment by SebastianMarzjan** [comment]
LG(TM)

**Pull request merged**